docs/HACKING.md

```diff
@@ -55,7 +59,10 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
 
     $ mkvirtualenv -a $PROJECT_HOME/goldstone-server goldstone-server
 
-   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original.
+   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original. Then:
```

Comment:
No, the intent is not to run python locally; rather, to start the container and use it.  That would come closer to guaranteeing a consistent environment between developers; and after packaging, between development and production.  We can add a helper, but basically, you can execute 

`docker exec -i -t goldstoneserver_gsappdev_1 python`

to get to an interactive python shell in the container, or if you want to add requirements on the fly, you could use:

`docker exec -i -t goldstoneserver_gsappdev_1 bash`

 to get to the command prompt.